pilot/pkg/serviceregistry/kube/controller/ambient/authorization.go

```diff
@@ -395,6 +396,11 @@ func convertPeerAuthentication(rootNamespace string, cfg, nsCfg, rootCfg *securi
 func convertAuthorizationPolicy(rootns string, obj *securityclient.AuthorizationPolicy) (*security.Authorization, *model.StatusMessage) {
 	pol := &obj.Spec
 
+	dryRun, convErr := strconv.ParseBool(obj.Annotations[annotation.IoIstioDryRun.Name])
+	if convErr != nil {
```

Comment:
```suggestion
	if convErr != nil {
		dryRun = false
```
I don't think Go defines the value of the first return value if there is an error (though in practice, it is `false`).